authenticate/handlers.go

```diff
@@ -282,6 +282,9 @@ func (a *Authenticate) SignOut(w http.ResponseWriter, r *http.Request) error {
 	// no matter what happens, we want to clear the session store
 	state.sessionStore.ClearSession(w, r)
 	redirectString := r.FormValue(urlutil.QueryRedirectURI)
+	if sru := a.options.Load().SignOutRedirectURL; sru != nil {
+		redirectString = sru.String()
```

Comment:
If both are supplied, I think query param should win?

Reply:
Ah yes.